MetaMask/metamask-mobile
# refactor(ramp): change settings title to buy & sell

## **Description**

This PR changes the Settings title for Ramp from "Buy" to "Buy & Sell" and adds tests for this view

## **Related issues**

Fixes:

## **Manual testing steps**

1. Go to Settings
2. Confirm the title change
## **Screenshots/Recordings**




### **Before**

<img width="200" src="https://github.com/MetaMask/metamask-mobile/assets/1024246/333dc674-b3dc-455f-9449-272f031d4160" />

### **After**

<img width="200" src="https://github.com/MetaMask/metamask-mobile/assets/1024246/03c969a5-6d52-449c-aa6f-4ced2117f886" />

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
